travel/templates/base.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+   <meta charset="utf-8">
+   <title>Principal Travel</title>
```

Comment:
Can we put a `title` content block in the title tags so we can change it per-page?